### PR TITLE
Crashing lambda ui

### PR DESCRIPTION
**Pull request by y-kkamil**
**Description**

Reworked solution from PR #885

### CODE REVIEW

File: lambda/src/app/lambdas/lambda-details/lambda-details.component.ts [hunk 1]
```diff
@@ -1231,6 +1233,8 @@ export class LambdaDetailsComponent implements OnInit, OnDestroy {
   }
 
   handleEnvEmitter($event): void {
+    this.lambda = this.lambdaDetailsService.initializeMissingLambdaContainers(this.lambda);
+
```
Comment (parostatkiem) [inline lambda/src/app/lambdas/lambda-details/lambda-details.component.ts]: Can't we simply `initializeMissingLambdaContainers` every time new Lambda data is received?
That would let us simplify `getEnvs()` a lot by doing just one property check.

File: lambda/src/app/lambdas/lambda-details/lambda-env/lambda-env.component.ts [hunk 2]
```diff
@@ -6,7 +6,7 @@ import {
   ViewChild,
   ViewContainerRef,
   Output,
-  EventEmitter,
+  EventEmitter
```
Comment (parostatkiem) [inline lambda/src/app/lambdas/lambda-details/lambda-env/lambda-env.component.ts]: Is it used?

File: lambda/src/app/lambdas/lambda-details/lambda-details.component.ts (hunk 1 above)
Comment (y-kkamil) [reply lambda/src/app/lambdas/lambda-details/lambda-details.component.ts]: fixed

**Comment by y-kkamil** [comment]
Another issue, which is somehow connected found: #914

**Pull request merged**